providers/amazon/docs/operators/mwaa.rst

```diff
@@ -76,6 +76,24 @@ In the following example, the task ``wait_for_dag_run`` waits for the DAG run cr
     :start-after: [START howto_sensor_mwaa_dag_run]
     :end-before: [END howto_sensor_mwaa_dag_run]
 
+.. _howto/sensor:MwaaDagRunSensor:
+
+Wait on the state of an AWS MWAA Task
+========================================
+
+To wait for a Task Instance within a DAG Run running on Amazon MWAA until it reaches one of the given states, you can use the
```

Comment:
We should clarify that this is intended to be used for sensing a task executed on a different MWAA environment.

also, the title is `Task` yet the description is `DAG Run`. It's not the same.
Does the sensor offer both capabilities?